ports/servo/build.rs

```diff
@@ -62,7 +62,7 @@ fn android_main() {
     println!("toolchain path is: {}", toolchain_path.to_str().unwrap());
 
     // Get the output directory.
-    let out_dir = env::var("OUT_DIR").ok().expect("Cargo should have set the OUT_DIR environment variable");
+    let out_dir = env::var("OUT_DIR").expect("Cargo should have set the OUT_DIR environment variable");
```

Comment:
This should probably also use `var_os`?

Reply:
This gets printed to stdout in addition to being used as a path, so unfortunately it needs to be validated as UTF-8.